code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -163,30 +166,17 @@ def verify_graph_with_cities(graph, start_city, end_city, cable_id):
 
     """
     all_points = set([p for p in graph])
-    if not start_city in graph:
-        closest_point, min_distance = min(
-            ((other_point, geodesic(start_city, other_point).kilometers)
-             for other_point in all_points),
-            key=lambda item: item[1]
-        )
-        threshold = 125
-        # Add edge if the closest distance is under the threshold and not in the current path
-        if min_distance < threshold:
-            add_edge(graph, start_city, closest_point)
-        # else:
-            # print(f"{start_city}, {closest_point} can not be merged with distance: {min_distance} for calbe {cable_id}")
-    if not end_city in graph:
-        closest_point, min_distance = min(
-            ((other_point, geodesic(end_city, other_point).kilometers)
-             for other_point in all_points),
-            key=lambda item: item[1]
-        )
-        threshold = 125
-        # Add edge if the closest distance is under the threshold and not in the current path
-        if min_distance < threshold:
-            add_edge(graph, end_city, closest_point)
-        # else:
-        #     print(f"{end_city}, {closest_point} can not be merged with distance: {min_distance} for calbe {cable_id}")
+    for city in [start_city, end_city]:
+        if not city in graph:
+            closest_point, min_distance = min(
+                ((other_point, geodesic(city, other_point).kilometers)
+                 for other_point in all_points),
+                key=lambda item: item[1]
+            )
+            threshold = 125
+            # Add edge if the closest distance is under the threshold and not in the current path
+            if min_distance < threshold:
+                add_edge(graph, city, closest_point)
```

Comment:
Raise exception here to make sure the rest of the code in this file can execute properly.

Reply:
fixed by adding an exception